client/packages/invoices/src/utils.ts

```diff
@@ -178,11 +179,21 @@ export const isOutboundDisabled = (
 
 export const isInboundDisabled = (inbound: InboundRowFragment): boolean => {
```

Comment:
Rename to `isOnHold`? or what does `isInboundDisabled` mean? At least some docs comment is needed

Reply:
we've used this naming elsewhere, and it doesn't necessarily mean 'on hold' so I think it is valid to keep 'isInboundDisabled'

just means that the user cannot interact / modify the invoice